ts/screens/wallet/payment/ScanQrCodeScreen.tsx

```diff
@@ -4,16 +4,24 @@
  */
 import { AmountInEuroCents, RptId } from "italia-pagopa-commons/lib/pagopa";
 import { ITuple2 } from "italia-ts-commons/lib/tuples";
-import { Container, Text, Toast, View } from "native-base";
+import { Button, Container, Text, Toast, View } from "native-base";
 import * as React from "react";
-import { Dimensions, ScrollView, StyleSheet } from "react-native";
+import {
+  Dimensions,
+  Linking,
+  Platform,
+  ScrollView,
+  StyleSheet
+} from "react-native";
+
 import QRCodeScanner from "react-native-qrcode-scanner";
 import { NavigationEvents, NavigationInjectedProps } from "react-navigation";
 import { connect } from "react-redux";
 
 import FooterWithButtons from "../../../components/ui/FooterWithButtons";
 import { CameraMarker } from "../../../components/wallet/CameraMarker";
 
+import AndroidOpenSettings from "react-native-android-open-settings";
```

Comment:
imports for external libraries are kept on top

Reply:
fixed